include/oneapi/dpl/pstl/utils.h

```diff
@@ -639,7 +641,7 @@ __pstl_lower_bound(_Acc __acc, _Size1 __first, _Size1 __last, const _Value& __va
         __it = __first;
         __cur = __n / 2;
         __it += __cur;
-        if (__comp(__acc[__it], __value))
+        if (__compareOp(__it))
```

Comment:
This line I would change to `if (std::invoke(__comp, std::invoke(__proj, __acc[__it]), __value))` with `__proj` being the new parameter for a projection.

Reply:
Will do.

Reply:
The PR https://github.com/uxlfoundation/oneDPL/pull/2425 has been merged into this branch.